pkg/operation/botanist/logging.go

```diff
@@ -35,7 +35,17 @@ import (
 
 // DeploySeedLogging will install the Helm release "seed-bootstrap/charts/loki" in the Seed clusters.
 func (b *Botanist) DeploySeedLogging(ctx context.Context) error {
-	if !b.Shoot.IsLoggingEnabled() {
+
+	// check if loki disabled in gardenlet config
+	var lokiDisabled bool
```

Comment:
Can you also name it `lokiEnabled` to make L48 easier readable? Also, shouldn't it then be initialized with `true`?